src/components/tracked-controls.js

```diff
@@ -18,7 +19,10 @@ module.exports.Component = registerComponent('tracked-controls', {
     controller: {default: 0},
     id: {type: 'string', default: ''},
     idPrefix: {type: 'string', default: ''},
-    rotationOffset: {default: 0}
+    rotationOffset: {default: 0},
+    // Arm model parameters, to use when not 6DOF. (pose hasPosition false, no position)
+    headElement: {type: 'selector'},
+    defaultHand: {type: 'string', default: 'right'}
```

Comment:
What I meant is that if the gamepad does not give us the hand we use `right` by default. The property is not exposed for the applications to change. The hand should be given by the gamepad API.